feature-libs/organization/account-summary/index.ts

```diff
@@ -0,0 +1 @@
+export * from './account-summary.module';
```

Comment:
if `index.ts` and `public_api.ts` both exist, it is better to remove `index.ts` (move its content to `public_api.ts`.). `index.ts` will not be generated by ng-packagar v14.